concept/ConceptManager.java

```diff
@@ -79,14 +78,17 @@ public ConceptManager(GraphManager graphMgr) {
         this.graphMgr = graphMgr;
     }
 
-    public FunctionalIterator<ConceptMap> conceptMaps(FunctionalIterator<VertexMap> vertexMap) {
-        return vertexMap.map(this::conceptMap);
+    public ConceptMap conceptMap(VertexMap vertexMap) {
+        return conceptMap(vertexMap, true);
     }
 
-    public ConceptMap conceptMap(VertexMap vertexMap) {
+    public ConceptMap conceptMap(VertexMap vertexMap, boolean useCachedVertices) {
```

Comment:
when we convert a concept map we can refresh the concepts to be the cached (ie canonical?) ones